src/App.js

```diff
@@ -4,21 +4,19 @@ import Board from "./components/Board";
 
 const App = () => {
   const [boardName, setBoardName] = useState("shonds-dubs");
```

Comment:
Used "shonds-dubs"when posting to boards right now. Please keep as is for your personal resume link though! I can create a board and can use this same code for a different "board" so it's one less thing to update overall..